[BLOCK] Tabs (#10)
moderntribe/moose-resources: block-library/tabs/README.md

## Example Usage

Both the editor view and FE view have been created fairly opinionated but both are very easily updated to match what would be required of your project. The editor view is different than the FE view in this case due to the difference in FE & Editor experience. 

- When you first add the block to the page (http://p.tri.be/i/SER4TN), the block will be populated with one tab. 
- New tabs can be added via the "Add New Tab" button. 
- Tabs can be removed by clicking the "X" icon within each tab title.
- Clicking each tab title should display the tab content for that tab title.
- Tabs can be reordered ONLY via the block list view at this time. We were not able to come up with a solution where we can click & drag to reorder the tabs within the block.
- Tab content can be entered by clicking each tab title and adding blocks to the tab content for that tab.

### Dependencies

The tabs block has a FE dependency on the `delegate` library (https://www.npmjs.com/package/delegate).

### Moose Implementation

Very simple implementation:

> 1. Add `delegate` as a project dependency in your `package.json` file
> 1. Run `npm install` to install the dependency 

Reviewer: ```suggestion
1. Run `npm i delegate` to add the required dependency. 
```
